docs/audit/IOS_DOCS_LEDGER_ALIGNMENT_AUDIT_2026-02-06.md

```
@@ -0,0 +1,152 @@
+### 3) Nutrition endpoint story drift + security risk (legacy alias)
+
+- **Repo truth (iOS client)**
+  - **Evidence command (stable):**
+
+```bash
+rg -n "api/nutrition/\\$\\{|api/nutrition/|not yet implemented" ios/PulsePlate/Models/NutritionData.swift -n
+```
+
+  - **Expected snippet:** iOS client currently builds path `api/nutrition/<date>` and contains a TODO claiming it is not implemented (docs must forbid this as SoT).
+
+- **Repo truth (backend)**
```

markdownlint-cli2 (0.20.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 0

(MD005, list-indent)